src/nxdoc/web-ui/web-ui-overview.md

```diff
@@ -226,15 +331,15 @@ Our Web UI is built itself with Polymer too. Goals were to not only ensure the s
 
 ### Web UI elements
 
-Everything in the web UI is built as a custom element with `<nuxeo-app>` as top level component. This application element acts as application data store / mediator storing context data as properties which are then forwarded through the hierarchy thanks to data binding. To learn more about data binding and its relation to the mediator pattern you can read the [article](http://www.nuxeo.com/blog/polymer-questions-to-bind-or-not-to-bind/) on our "[Polymer](http://www.nuxeo.com/blog/tag/polymer/)" series.
+Everything in the web UI is built as a custom element with `<nuxeo-app>` as top level component. This application element acts as application data store / mediator storing context data as properties which are then forwarded through the hierarchy thanks to data binding. To learn more about data binding and its relation to the mediator pattern you can read the [article](http://www.nuxeo.com/blog/polymer-questions-to-bind-or-not-to-bind/).
 
 `<nuxeo-app>` relies on our data elements (`<nuxeo-document>`, `<nuxeo-resource>`, etc..) to fetch application wide data and it is in charge of refreshing this data when changes are made. We are not relying on two way binding in order to keep the data flux manageable thus parent elements act as data stores and children fire events to notify of changes causing data to be refreshed and then propagated down through the DOM tree. Alternative would be to rely on a custom event / action system like many framework do but, although this would avoid the cost inherent to event bubbling, it would make our elements dependent on the the existence of such as bus and introducing framework lock-in.
 
 ![]({{file name='Web UI document page.png'}} ?w=959,h=614,border=true)
 
 The drawer's content is managed by `<iron-pages>` which toggles between its children depending on current selection in the left icon menu of the UI.
 
-The main area is reserved for displaying the current page's content. Our router relies on [page.js](https://visionmedia.github.io/page.js/) and is in charge of parsing the URL and setting the current page and context data in `<nuxeo-app>` accordingly.
+The main area is reserved for displaying the current page's content. Our router relies on [Nuxeo Router Behavior](https://doc.nuxeo.com/nxdoc/web-ui-routing/) and it is in charge of parsing the URL and setting the current page and context data in `<nuxeo-app>` accordingly.
 
 ![]({{file name='Web UI search page.png'}} ?w=959,h=614,border=true)
 
```

Comment:
## Accessibility

Nuxeo is committed to making Web UI accessible and keeping a high level of quality on that front. Our target is the WCAG standard level AA. Our current level of support for accessibility is controlled regularly, and we publish the results using the standard section 508 Compliance (VPAT) document. Its latest iteration can be found below.

- **Date**: February, 2020
- **Product**: Nuxeo Web UI
- **Link**: [Voluntary Product Accessibility Template for Web UI]({{file name='Nuxeo Web UI-VPAT 2020.pdf'}}).

Reply:
@semisse I updated the section below (section 508) to make it more search / SEO friendly. Can you please help me incorporate that? :pray: